src/test/java/com/microsoft/sqlserver/jdbc/fedauth/FedauthWithAE.java

```diff
@@ -128,7 +121,7 @@ public void testFedAuthWithAE_AKV() throws SQLException {
             dropCMK(stmt, cmkName3);
             setupCMK_AKVOld(cmkName3, stmt);
 
-            createCEK(cmkName3, setupKeyStoreProvider_AKVOld(), stmt, keyIDs[0]);

```

Comment:
This test should not be removed, it is the only test thats actually using `SQLServerColumnEncryptionAzureKeyVaultProvider(authenticationCallback)`.

Reply:
SQLServerColumnEncryptionAzureKeyVaultProvider(authenticationCallback) is deprecated for removal in the next stable release, so I think we can remove calling this test method.